src/client/components/compendium/Compendium.tsx

```diff
@@ -85,7 +85,7 @@ export function Compendium() {
               multiple
               onChange={async (e) => {
                 const files = Array.from(e.target.files ?? []);
-                const jsons = await Promise.all(
+                const JSONs = await Promise.all(
```

Comment:
According to our AC discussion, this would probably need to be lowercased to be consistent, maybe to `fileContents` if `jsons` is a spelling error?